Caltech-IPAC/firefly
# DM-12305: HIPS: begin proof of concept

DM-12305: HIPS: begin proof of concept

   - scrolling infinite sky
   - create a transform
   - working on tiling
   - More optimized drawing
   - display using properties
   - transform clean up
   - clean up zoom
   - drawers: partially working: grid, working: north up,catalogs
   - more work - center, grouping
   - work on coverage with hips
   - radius size better,
   - displayed fov fixed
   - switch surveys on the fly


_to tests:_

- http://localhost:8080/firefly/firefly-dev.html
- choose `test searches` then `hips`
- choose any  target and any dataset

_to tests coverage:_

- http://localhost:8080/firefly
- search for a catalog
- coverage should be a hips dataset

_files to focus on:_
There are too many files to review.

Focus on:
- HIPSGrid.js
- HIPSUtils.js
- ImagePlotCntlr.js
- PlotViewUtil.js
- HiPSTileDrawer.js
- HiPSTileDraw.js
